src/main/java/seedu/dietbook/Manager.java

```diff
@@ -1,15 +1,24 @@
 package seedu.dietbook;
 
+import seedu.dietbook.command.*;
```

Comment:
Perhaps you might want to make a specific imports rather than wildcard imports.

Reply:
Understandable but the reason for all is because all command is used so there would be like 11 import statements if i am specific? what do you think?